bundles/specmate-persistency-cdo/src/com/specmate/persistency/cdo/internal/HistoryProviderImpl.java

```diff
@@ -79,9 +118,27 @@ protected void changedObject(CDOID id, EStructuralFeature feature, EChangeKind c
 			if (!id.equals(this.cdoId)) {
 				return;
 			}
+			
+			//Not interested in containment features that changed. 
+			if(feature instanceof EReference) {
+				EReference ref = (EReference) feature;
+				if(ref.isContainment())
+					return;
+			}
+			
 			Change change = HistoryFactory.eINSTANCE.createChange();
 			change.setFeature(feature.getName());
-			change.setValue(newValue.toString());
+			 
+			if(newValue != null)
+				change.setNewValue(newValue.toString());
+			else {
+				//FIXME when elements are deleted, the deltaprocessor does not receive them as detached objects but rather as

```

Comment:
If possible, avoid FIXME comments. In my experience, these  things are never fixed in the end. Or is there a good reason for this here?

Reply:
I added this to get attention in the review. It worked :). 
My idea was that FIXMEs are removed before merged to develop.